README.md

```diff
@@ -176,11 +171,9 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 - ![v3] [vite-plugin-vuedoc](https://github.com/JasKang/vite-plugin-vuedoc) - Markdown Code block as Vue Preview components.
 - ![v3] [vite-svg-loader](https://github.com/jpkleemans/vite-svg-loader) - Load SVG files as Vue components.
 
-#### SSR / SSG
+#### SSG
 
 - ![v3] [vite-ssg](https://github.com/antfu/vite-ssg) - Server-side generation.
-- ![v3] [vite-ssr](https://github.com/frandiox/vite-ssr) - Server-side rendering.
-- ![v3] [vitedge](https://github.com/frandiox/vitedge) - Edge-side rendering with fullstack utilities.
 
```

Comment:
If we extract SSR into another section, maybe it's better to add a line saying that "Refer to SSR section for SSR supports"

Reply:
What do you mean?

Reply:
Like, I am look into the Vue section and trying to find the SSR support for Vue. But now we move it to another section, which might confuse ppl like "Oh Vite does not support SSR for Vue?" so some sort of the things.

Reply:
Ah, I see.

Reply:
Done.

I've decided to remove the SSR Boilerplates section because the whole idea here is to let users know about higher-level alternatives to Vite's native SSR API; boilerplates are not contributing here.